ocs_ci/ocs/disruptive_operations.py

```diff
@@ -0,0 +1,59 @@
+import logging
+import time
+
+from ocs_ci.ocs.node import get_typed_nodes, get_node_name
+from ocs_ci.ocs.resources.pod import (get_ocs_operator_pod, get_pod_node)
+from ocs_ci.ocs.platform_nodes import PlatformNodesFactory
+from tests.helpers import wait_for_ct_pod_recovery
+
+log = logging.getLogger(__name__)
+
+
+def get_ocs_operator_node_name():
+    """
+    Getting node's name that running ocs-operator pod
+
+    Returns:
+        str: node's name that running ocs-operator pod
+
+    """
+    ocs_operator_pod = get_ocs_operator_pod()
+    log.debug(f"ocs operator pod info: {ocs_operator_pod}")
+    ocs_operator_node = get_pod_node(ocs_operator_pod)
+
+    return get_node_name(ocs_operator_node)
+
+
+def worker_node_shutdown(abrupt):
+    """
+    Shutdown worker node that running ocs-operator pod
+
+    Args:
+        abrupt: bool: True if abrupt shutdown, False for permanent shutdown
+
```

Comment:
You can also add Raises: section with AssertionError and describe when it's raised

Reply:
added